src/base-table.js

```diff
@@ -0,0 +1,488 @@
+ */
+
+/**
+ * @type {Record<CodecName,CodecNumber>}
```

Comment:
this will cause the error you are seeing, use the types in types.ts instead

Reply:
If I use types.ts, then it will complain it's a recursive dependency, because types.ts gets the types from base-table. I'm separating them now.

Reply:
The commit I just pushed works!

Reply:
ahhh right thats why i just ignore this type in multihash https://github.com/multiformats/js-multihash/blob/master/src/constants.js#L16 😱

Reply:
Pushed